api_docs/changelog.md

```diff
@@ -20,6 +20,17 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 201**
+
+* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events):
+  `realm_linkifiers` events now send linkifier objects sorted by the order in
+  which they are applied when linkifying messages and topics.
```

Comment:
Technically, they have always been in the order they'll be applied -- the change is that we're allowing you to change that order from "sorted by ID", right? I think that means that there's no actual API change for these endpoints?

Reply:
Yeah. Should we mention here that the arrays sent for these events are no longer only sorted by ID?

Reply:
Just saw the other comments in the API document. I think the right thing to do would be just deleting this bullet here.

Reply:
Also removed the one for `GET /realm/linkifiers`.